### ISSUE TITLE
Pretrained weight availability

### ISSUE THREAD

**Issue by ZhiyuanChen** [Shujun-He/RibonanzaNet#2]
Hi there

I'm recently developing [MultiMolecule](multimolecule.danling.org), and we have included many pre-trained foundation models in our hub. 

I believe RibonanzaNet can be an excellent addition to our library. But I'm having troubles to locate the pre-trained weights. 

Can I ask how can I access the pretrained weights of RibonanzaNet?

**Comment by Shujun-He** [comment]
Hi Zhiyuan,

You can find the weights on Kaggle at
https://www.kaggle.com/datasets/shujun717/ribonanzanet-weights

*Description:*
Dataset containing RibonanzaNet pretrained and finetuned weights

RibonanzaNet.pt: pretrained checkpoint trained with chemical mapping data
RibonanzaNet-Deg.pt: finetuned checkpoint trained with RNA degradation data
from OpenVaccine
RibonanzaNet-SS.pt: finetuned checkpoint (v0.9) trained with pdb secondary
structure data
RibonanzaNet-Drop.pt: finetuned checkpoint trained with 2A3 and DMS
Illumina sequencer reads from Ribonanza

Thanks,
Shujun

**Comment by ZhiyuanChen** [comment]
Thank you for your prompt reply! And apologies for this late follow up, I was extremely busy in the past few months. 

I noted that the training pipeline of the RibonanzaNet is as follows: 

```markdown
1. An initial model was trained using sequences that have either or both 2A3/DMS profiles
with signal to noise ratio above 1.0 (214,831 training sequences) over 30 epochs, which took 11
hours on 10xL40S GPUs. 

2. A second model ensembled predictions for the remaining (noisy)
training data from the top 3 Kaggle models as pseudo-labels (563,796 sequences), and pre-
trained RibonanzaNet for 30 epochs with these pseudo-labels and a flat learning rate of 0.001,

3. followed by 10 epochs of training solely on true labels of training sequences that have one
profile of 2A3/DMS with high signal-to-noise ratio, following a cosine learning rate schedule
that annealed learning rate to 0. 
```

Can I please confirm what is the training objectives of the initial training? Are they Masked Language Modelling?
